conda/meta.yaml

```diff
@@ -56,5 +44,5 @@ requirements:
     - python
     - setuptools
     - setuptools_scm
-    - ts-salobj
-    - ts-simactuators
+    - ts-idl {{ idl_version }}
+    - ts-salobj {{ salobj_version }}
```

Comment:
Can you tell me where these values are set? I'm just curious.

Reply:
I suppose somewhere in the Docker environment. Rob asked me to add that so I did :)